src/OrbitGl/App.cpp

```diff
@@ -2711,6 +2711,16 @@ void OrbitApp::SetMaxLocalMarkerDepthPerCommandBuffer(
       max_local_marker_depth_per_command_buffer);
 }
 
+void OrbitApp::SetEnableAutoFrameTrack(bool enable_auto_frame_track) {
+  // if the option is true, Orbit will try to add the default frame track as soon as possible. This
+  // might fail because a user can start a capture before the needed symbols are downloaded, so we
+  // are additionaly saving the state for the future.
+  if (enable_auto_frame_track) {
+    std::ignore = AddDefaultFrameTrackOrLogError();
```

Comment:
Is using `std::ignore` really the best option here? It always seems a bit strange to use something like this if you have a mechanism that forces return values to be used.

Reply:
The function is returning `Future<void>`, we're just not waiting for the future to be finished. We had a discussion in https://github.com/google/orbit/pull/3954#discussion_r923549530 where @antonrohr proposed to use either `Future<void>` or either `void.` Maybe I should return void if it's confusing.

Reply:
Ah, I see, is there any point in using `Future<void>` instead of `void`? I would probably prefer simply `void` to avoid the std::ignore here, simply for readability and avoiding confusion.

Reply:
TBH, not. I will change it in a PR next week.